_posts/en/newsletters/2019-10-02-newsletter.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Bitcoin Optech Newsletter #66'
+permalink: /en/newsletters/2019/10/02/
+name: 2019-10-02-newsletter
+slug: 2019-10-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter notes a proposed BIP that would allow nodes to
+implement the erlay transaction relay protocol, announces full
+disclosure of vulnerabilities that affected earlier versions of LN
+implementations, and links to a transcript of Optech's recent schnorr
+and taproot workshop.  Also included is the description of several
+notable changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Draft BIP for enabling erlay compatibility:** a draft [BIP for
+  transaction relay through set reconciliation][bip-reconcil] has been
+  [posted][reconcil post] to the Bitcoin-Dev mailing list by [erlay][]
+  co-author Gleb Naumenko.  Currently, Bitcoin nodes send each of their
+  peers the txids for all newly-seen transactions, resulting in each
+  node receiving many duplicate txid announcements.  This can be quite
+  bandwidth inefficient for Bitcoin's tens of thousands of nodes and
+  several hundred thousand transactions a day.  As [described
+  previously][erlay paper], minisketch-based set reconciliation allows
+  nodes to instead send a *sketch* of a set of txids that can be combined
```

Comment:
nit: to instead send -> to send (split infinitive and instead doesn't add extra meaning.)